OsmAnd/src/net/osmand/plus/osmedit/utils/SecUtils.java

```diff
@@ -0,0 +1,296 @@
+package net.osmand.plus.osmedit.utils;
```

Comment:
All imported classes put in 1 special package
package org.openplacereviews.opendb.import;

Reply:
All copy / paste code there